scripts/roles/todd/tasks/main.yml

```diff
@@ -0,0 +1,123 @@
+---
+- include_vars: toddvars.yml
+
+- name: Set PATH and GOPATH and for Vagrant user and shell profiles
+  lineinfile:
+    dest: "{{ item.file }}"
+    state: "present"
+    regexp: "^export {{ item.varname }}"
+    line: "{{ item.varval }}"
+  with_items:

```

Comment:
I would move everything from lines 11-14 to your `todd/vars/toddvars.yml` and maybe format them as below:
```
todd_path_exports:
  - file: '/home/vagrant/.profile'
    varname: 'GOPATH'
    varval: 'export GOPATH={{ gopath }}'
  - file: '/home/vagrant/.profile'
    varname: 'PATH'
    varval: 'export PATH={{ path }}'
  - file: '/home/vagrant/.bashrc'
    varname: 'GOPATH'
    varval: 'export GOPATH={{ gopath }}'
  - file: '/home/vagrant/.bashrc'
    varname: 'PATH'
    varval: 'export PATH={{ path }}'
```
Then change line 10 to:
```
with_items: '{{ todd_path_exports }}'
```
Also, where are `gopath` and `path` variables defined for the `varval` variables?